.circleci/config.yml

```diff
@@ -1158,11 +1158,11 @@ jobs:
       - run:
           name: Install Geth
           command: |
-            wget https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.13.4-3f907d6a.tar.gz
```

Comment:
This is faster than `go install`?

Reply:
i don't know why it was done this way.

Reply:
I think it's faster, it's the pre-built package from https://geth.ethereum.org/downloads

Reply:
Interestingly, in the ci-builder image itself, we use `go install`